content/post/data-dies-in-darkness.md

```diff
@@ -0,0 +1,73 @@
+---
+author: Michael Kaminsky
+title: "Data Dies in Darkness"
+date: "2018-02-25"
+tags: 
+ - analytics
+ - data quality
+ - org
+---
+
+# Data Dies in Darkness 
+### Building the Data Quality Flywheel
+
+
+The fastest way to doom an analytics team (and any hope of building a data-driven organization) is to present data and analyses that are often flawed or inconsistent. When people don't believe they can trust the data, they'll stop using them (and, if you're an analytics leader, you might be soon looking for a new job).
+
+There are two techniques I rely on for ensuring data and analytical quality:
+1. Analysis audit and code review
+2. The internal data quality flywheel
+
+While the first technique (good analysis review) is very important, in this post I'm going to focus on the second which is an incredibly powerful tool for thinking about not only adding value as a data and analytics team, but will pay ongoing dividends in data quality for years to come.
+
+When people talk about data quality, they often describe the *symptoms* of data quality issues. Like:
+* Reporting numbers changing over time
+* Unclear definitions of metrics
+* Inconsistent numbers across teams or reports
+
+These symptoms are generally driven by a few root issues:
+* Lack of a centrol repository of conformed data (single source of truth)
+* Disagreement or misalignment about how business phenomena or KPIs ought to be measured
+* Unnoticed "data drift" as business-logic assumptions or data integrations get stale
```

Comment:
I think we should get in the habit of linking or footnoting related comment if we're not going to go into an important idea like this  

https://streamsets.com/reports/data-drift/

Reply:
I didn't know this was a commonly used term! Adding a link